scripts/opdracht1.js

```diff
@@ -0,0 +1,44 @@
+// #### Opdracht 1 - Array Methoden//
+// * **Opdracht 1a:** Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de
+// console.
+//
+// Hier gebruik ik de map methode voor. Eerst een variabel aanmaken om het in op te kunnen slaan. Vervolgens de parameter een soortgelijke naam geven zodat we daarmee de waardes van de array terugkrijgen. Door middel van puntnotatie kun je keys in objecten aanspreken. Met return functie nieuwe array maken.
+
+const tvTypeNames = inventory.map((inventories) => {
+    return inventories.name;
+})
+
+console.log(tvTypeNames);
+
+// * **Opdracht 1b:** Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht
+// zijn. Log de uitkomst in de console.
+//
+// Hier moeten we filteren op uitverkocht dus is de filtermethode geschikt. Eerst een variabel aanmaken om het in op te kunnen slaan. Parameter weer benoemen zodat we de waardes terug kunnen krijgen. De keys die we nodig hebben om volledig uitverkocht terug te krijgen is originalStock - sold === 0. Return dan true.
+
+const tvSoldOut = inventory.filter((inventories) => {
+    return inventories.originalStock - inventories.sold === 0;
+})
```

Comment:
Dit zou je nog wat korter kunnen schrijven
```suggestion
const tvSoldOut = inventory.filter(inventories => inventories.originalStock === inventories.sold);
```